packages/venia-ui/lib/components/CheckoutPage/PaymentInformation/paymentInformation.css

```diff
@@ -5,14 +5,31 @@
 }
 
 .container {
+    display: grid;
+    grid-template-columns: 1fr;
+    gap: 2rem;
+}
+
+.payment_info_container {
     min-height: 10rem;
     border: 1px solid rgb(var(--venia-border));
     border-radius: 5px;
     padding: 1.5rem 1rem;
 }
 
-.heading {
-    line-height: 1.25em;
-    border-bottom: 1px solid rgb(var(--venia-border));
-    padding-bottom: 2rem;
+.price_adjustments_container {
+    grid-column: 1 / span 1;
+}
+
+.review_order_button {
+    grid-column: 1 / span 1;
+    composes: root_highPriority from '../../Button/button.css';
```

Comment:
In a list of properties, `composes` should appear first.

Reply:
Good catch, I'll change this wherever I see it in the checkout page directory.